utilities/mswyw.py

```diff
@@ -101,18 +109,29 @@ def main():
         overrides = json.loads(arguments.get("--overrides", "{}"))
         mswyw_score = compute_formula(arguments.get("--calcProvider"), ms_runtime_data, formula_coefficients, overrides)
         script_end_time = datetime.datetime.now()
-        print("\r\n--------------------------------------------------")
-        print("Sampling Start time: %sZ" % sampling_start_time.isoformat())
-        print("Sampling End time:   %sZ" % sampling_end_time.isoformat())
-        print("Instances:")
-        for runtime_data in ms_runtime_data:
-            print(runtime_data)
-        print("\r\n--------------------------------------------------")
-        print("Started : %s" % str(script_start_time))
-        print("Finished: %s" % str(script_end_time))
-        print("Total: %s" % str(script_end_time - script_start_time))
-        print("mswyw score: %s" % str(mswyw_score))
-        print("--------------------------------------------------")
+
+        if arguments.get("--jsonOutput"):
+            information = {"score": mswyw_score, "instance_count": len(ms_runtime_data)}
+
+            information["name"] = provider_params['elastic.APPS'] if arguments.get("--runtimeProvider") == "elastic" else provider_params["nrelic.APPID"]
```

Comment:
That was the way I found to have the name of the application on the exported data. I could not realize how to dot it without knowing the plugin which is being used to generate app data. Could please point me out how to properly get this information?
The idea here is to export the app name used to filter the data and since the origin of this data is from a plugin configuration I thought I could use it.